docs/open_source/README.md

```diff
@@ -18,15 +18,14 @@ We have several ways of pushing content to external developers:
 
 * Node Modules: the code in an installable form (See Releases section)
 * GitHub Documentation: Explain how to use the Node modules
-* Pivotal Backlog: Detail our priorities for fixing bugs and completing feature requests
+* GitHub Milestones: Detail our priorities for fixing bugs and completing feature requests
 * Catalog: Visualize our components (See Releases section)
 
 GitHub is where we do most of our communication with external developers. The
 GitHub documentation is critical to our external developers understanding how
 to use our Node modules. GitHub is also where they can file bugs and feature
-requests. External developers see these issues prioritized on our Pivotal
-tracker. External developers can view the Backlog and see the time estimate
-until their bug is fixed, or their feature request is complete.
+requests. External developers see these issues prioritized in our release milestones
+on GitHub.
 
 ## Release Cop
```

Comment:
Please fix the link or unlink "Untriaged Issues" bullet in Release Cop section.

Reply:
Oh nice catch. I removed the Untriaged Issues bullet, then linked the Triage Process and Release Process bullets to their docs in this folder.